lab-taylor/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('runTests', () => {
+  gulp.src(__dirname + '/test/*.js')
+  .pipe(mocha());
+});
+
+gulp.task('runLint', () => {
+  return gulp.src(['**/*.js','!node_modules/**'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+
+gulp.task('watch', () => {
+  gulp.watch(['**/*.js', '!node_modules/**'], ['runTests', 'runLint']);
+});
+
+gulp.task('default', ['runTests', 'runLint', 'watch'], () => {
```

Comment:
Just fyi if you refactor this, you don't need the anonymous function at the end of this task.
gulp.task('default', ['runTests', 'runLint', 'watch']);
should work fine too :D